Source/ProgressionSystemRuntime/Public/Widgets/PSOverlayWidget.h

```diff
@@ -13,21 +14,45 @@ UCLASS()
 class PROGRESSIONSYSTEMRUNTIME_API UPSOverlayWidget : public UUserWidget
```

Comment:
Please add comment above PSOverlayWidget as everything in .h we want to explain for others: what the main feature it provides, when it's displayed (Menu)

Reply:
done with: https://github.com/h4rdmol/ProgressionSystem/pull/20/commits/21542488cb1ec290a630a20085cc0952fa696061